### ISSUE TITLE
Decide whether to use Django All Auth or Python Social Auth for sign ins.

### ISSUE THREAD

**Issue by brittdawn** [brittdawn/django-webstore#46]

**Comment by rwisecar** [comment]
Python Social Auth is built off a Django All Auth base, right? But with enhanced social media logins and API docs? I'd be happy with either, but PSA sounds pretty cool

**Comment by brittdawn** [comment]
Agreed. Let's dive into the PSA docs tomorrow for a bit to decide how we want to integrate with the invoice app. :thumbs up:
